packages/cli/src/lib/utils/process.ts

```diff
@@ -73,3 +74,18 @@ export async function spawnProcessOutput(
     });
   });
 }
+
+export function spawnProcessPTY(
+  command: string,
+  args: string[],
+  options: SpawnOptions = {}
+) {
+  const ptyProcess = ptySpawn(command, args, {
+    name: 'xterm-color',
+    cols: process.stdout.columns,
+    rows: process.stdout.rows,
+    cwd: options.cwd || process.cwd(),
```

Comment:
Use `??` instead of `||`. `cwd:''` is valid IMO

Reply:
good catch